third_party/move/move-compiler-v2/transactional-tests/tests/control_flow/sorter.exp

```diff
@@ -0,0 +1,3 @@
+processed 2 tasks
+
+==> Compiler v2 delivered same results!
```

Comment:
I thought it was clear that all those transactional tests first execute the test with the v1 compiler and record the output, then execute the v2 compiler and compare the results, and finally print this message if the results are equal. This is the entire point of this test suite. The PR for this was nearly a week in review.

Reply:
Sorry, I had assumed that you were comparing with the v1 execution which is already happening, and whose output is already present in a corresponding .exp file over in /move-compiler/ somewhere.  I failed to look at the test driver on that PR since I had assumed someone else knew more about it.  Now I get it.